packaging/wheel/publish.sh

```diff
@@ -5,4 +5,35 @@ version=$(./genversion.sh --print-only)
 version=${version#v}
 echo $version > bindings/python/VERSION
 rm -r dist
-python3 setup.py sdist
+
+# Determine if wheel.bdist_wheel is available for wheel.bdist_wheel in setup.py
+python3 -c 'import wheel' &> /dev/null
+wheel_available=$?
+if [ "${wheel_available}" -ne "0" ]; then
+    python3 -m pip install wheel
+    res=$?
+fi
+unset wheel_available
+
+if [ "$res" -ne "0" ]; then
+    echo "ERROR: Unable to find wheel and unable to install wheel with '$(command -v python3) -m pip install wheel'."
+    echo "       Please ensure wheel is available to $(command -v python3) and try again."
+    exit 1
+fi
+
+# Determine if build is available
+python3 -c 'import build' &> /dev/null
+build_available=$?
+if [ "${build_available}" -ne "0" ]; then
+    python3 -m pip install build
+    res=$?
+fi
+unset build_available
+
+if [ "$res" -ne "0" ]; then
+    echo "WARNING: Unable to find build and unable to install build from PyPI with '$(command -v python3) -m pip install build'."
+    echo "         Falling back to building sdist with '$(command -v python3) setup.py sdist'."
+    python3 setup.py sdist
+else
+    python3 -m build --sdist .
+fi
```

Comment:
As attempting to use the deprecated `python setup.py sdist` with a modern `setuptools` is going to warn you to not and to use `build` (c.f. Issue #1579)

```
```pytb
/usr/local/venv/lib/python3.9/site-packages/setuptools/command/install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
/usr/local/venv/lib/python3.9/site-packages/setuptools/command/easy_install.py:156: EasyInstallDeprecationWarning: easy_install command is deprecated. Use build and pip and other standards-based tools.
  warnings.warn(
```

then attempt to use `build` to perform the sdist build. If it is unable to get `build` from PyPI then fall back to `python setup.py sdist` with a warning.